### PR TITLE
Replace RangeID with MetaRangeID

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

**Review by nopcoder — APPROVED** [review]

File: graveler/testutil/fakes.go [hunk 1]
```diff
@@ -11,66 +11,66 @@ const DefaultBranchID = graveler.BranchID("master")
 
 type AppliedData struct {
 	Values  graveler.ValueIterator
-	RangeID graveler.RangeID
+	RangeID graveler.MetaRangeID
```
Comment (guy-har) [inline graveler/testutil/fakes.go:14]: change RangeID to MetaRangeID

File: graveler/testutil/fakes.go [hunk 2]
```diff
@@ -11,66 +11,66 @@ const DefaultBranchID = graveler.BranchID("master")
...
 }
 
 type CommittedFake struct {
 	Value         *graveler.Value
 	ValueIterator graveler.ValueIterator
 	diffIterator  graveler.DiffIterator
 	Err           error
-	RangeID       graveler.RangeID
+	RangeID       graveler.MetaRangeID
 	AppliedData   AppliedData
```
Comment (guy-har) [inline graveler/testutil/fakes.go:23]: change RangeID to MetaRangeID

File: graveler/testutil/fakes.go [hunk 3]
```diff
@@ -153,7 +153,7 @@ func (s *StagingFake) ListSnapshot(_ context.Context, _ graveler.StagingToken, _
 type AddedCommitData struct {
 	Committer string
 	Message   string
-	RangeID   graveler.RangeID
+	RangeID   graveler.MetaRangeID
 	Parents   graveler.CommitParents
```
Comment (guy-har) [inline graveler/testutil/fakes.go:157]: change RangeID to MetaRangeID

File: graveler/uncommitted_diff_iterator.go [hunk 4]
```diff
@@ -11,13 +11,13 @@ type uncommittedDiffIterator struct {
 	committedManager CommittedManager
 	list             ValueIterator
 	storageNamespace StorageNamespace
-	rangeID          RangeID
+	rangeID          MetaRangeID
 	value            *Diff
```
Comment (guy-har) [inline graveler/uncommitted_diff_iterator.go:15]: change RangeID to MetaRangeID

File: graveler/uncommitted_diff_iterator.go [hunk 5]
```diff
@@ -11,13 +11,13 @@ type uncommittedDiffIterator struct {
...
 	err              error
 	ctx              context.Context
 }
 
-func NewUncommittedDiffIterator(ctx context.Context, manager CommittedManager, list ValueIterator, sn StorageNamespace, rangeID RangeID) DiffIterator {
+func NewUncommittedDiffIterator(ctx context.Context, manager CommittedManager, list ValueIterator, sn StorageNamespace, rangeID MetaRangeID) DiffIterator {
 	return &uncommittedDiffIterator{
```
Comment (guy-har) [inline graveler/uncommitted_diff_iterator.go:21]: change RangeID to MetaRangeID

**Pull request merged**